root/types.js

```diff
@@ -557,7 +557,7 @@ declare type GroupedOptionsT = $ReadOnlyArray<{
 
 declare type InstrumentCreditsAndRelTypesRoleT = {
   +instrumentCreditsAndRelTypes?: {
-    +[string]: $ReadOnlyArray<string>,
+    +[entityGid: string]: $ReadOnlyArray<string>,
```

Comment:
This one can be found at https://github.com/metabrainz/musicbrainz-server/blob/974e52b1fb7619b6fdcf2eb73de53771a071b9fe/root/static/scripts/common/components/TagEditor.js#L401